File: Comunicação de Sistemas heterogêneos/Cilente/src/tsi/webservices/cliente/model/Turma.java
package tsi.webservices.cliente.model;

import java.util.ArrayList;
import java.util.List;

public class Turma {
	private Long id;
	private Integer ano;
	private String curso;
	private ArrayList<Aluno> alunos;
	
	public Turma() {
		alunos = new ArrayList<>();
	}
	
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public Integer getAno() {
		return ano;
	}
	public void setAno(Integer ano) {
		this.ano = ano;
	}
	public String getCurso() {
		return curso;
	}
	public void setCurso(String curso) {
		this.curso = curso;
	}
	public Aluno[] getAlunos() {
		return alunos.toArray(new Aluno[0]);
	}
	public void setAlunos(Aluno[] alunos) {
		for (Aluno aluno : alunos) {
			this.alunos.add(aluno);
		}
	}
	public void setAlunos(List<Aluno> alunos) {
		if (alunos instanceof ArrayList<?>)
			this.alunos = (ArrayList<Aluno>) alunos;
		else
			setAlunos(alunos.toArray(new Aluno[0]));
	}
	public void addAluno(Aluno aluno) {
		alunos.add(aluno);
	}
}

File: Comunicação de Sistemas heterogêneos/Servidor/turma.py

from aluno import Aluno

class Turma:
    def setId(self, id):
        self.__id = id
        self.__alunos = []

    def setAno(self, ano):
        self.__ano = ano

    def setCurso(self, curso):
        self.__curso = curso

    def setAlunos(self, alunos):
        self.__alunos = alunos

    def getId(self):
        return self.__id

    def getAno(self):
        return self.__ano

    def getCurso(self):
        return self.__curso

    def getAlunos(self):
        return self.__alunos

    def addAluno(self, aluno):
        self.__alunos.append(aluno)
